snyk-import/main.ts

```diff
@@ -0,0 +1,100 @@
+// deno-lint-ignore-file no-explicit-any
+
+import { ApiVersion, PhylumApi } from "phylum";
+import { parseArgs } from "https://deno.land/std@0.211.0/cli/parse_args.ts";
+import {
+  getOrgs,
+  getProjectDependencies,
+  getProjects,
+  Project,
+} from "./snyk-api.ts";
+import { setToken, snykToken } from "./token.ts";
+
+function usage() {
+  console.log(
+    "phylum snyk-import [--token SNYK_TOKEN] [--no-save-token] [--group phylum_group]",
```

Comment:
~~It seems like `--token` is the only way to specify the token? Usually a CLI argument for an API token isn't ideal since that makes it visible in the cmdline. Should we maybe support an environment variable or allow reading it from stdin?~~

Saw that you do allow the user to interatively login and specify their token, but I don't want to delete this comment entirely since the drawbacks of CLI arg versus env variable/stdin still exist.